### PR TITLE
added alert while signing out

### LINKED ISSUE #34: Alert to confirm sign out
Right now, the user gets signed out as soon as he clicks the sign out button on appbar. It would be better to show an alert dialog and confirm if the user actually wants to sign out.

### PR DESCRIPTION
**Pull request by chandansgowda**
Closes #34

### CODE REVIEW

File: lib/screens/tabs_screen.dart [hunk 1]
```diff
@@ -40,9 +39,31 @@ class _TabsScreenState extends State<TabsScreen> {
...
+                  showDialog(
+                      context: context,
+                      builder: (_) {
+                        return AlertDialog(
+                          title: Text("Confirm Sign Out?"),
+                          actions: [
+                            ElevatedButton(
+                              onPressed: () async{
+                                GoogleSignIn googleSignIn = GoogleSignIn();
+                                await googleSignIn.disconnect();
```
Comment (letsintegreat) [inline lib/screens/tabs_screen.dart:51]: This line is returning an exception when the sign in method is not Google. Figure out if there is a way to find the sign in method, or just add this line in try catch block.

**Review by letsintegreat — CHANGES_REQUESTED** [review]
Great work! Just one change.

File: lib/screens/tabs_screen.dart (hunk 1 above)
Comment (chandansgowda) [reply lib/screens/tabs_screen.dart]: I have corrected it now. Please review.

**Comment by letsintegreat** [comment]
@chandansgowda What if a user enters a Gmail ID while signing up with via email method? Will this still work correct?

**Comment by chandansgowda** [comment]
@letsintegreat It'll work correctly

**Review by letsintegreat — APPROVED** [review]
Alright then, thanks for contributing!

**Pull request merged**